evm/README.md

```diff
@@ -166,9 +180,66 @@ $ make fix-fmt
 $ cd evm && forge snapshot
 ```
 
-### Contract Deployment
-TODO
+### Cast
+
+```shell
+$ cast <subcommand>
+```
+
+### Help
+
+```shell
+$ forge --help
+$ anvil --help
+$ cast --help
+```
+
+### Deploy Wormhole NTT
+
+#### Environment Setup
+
+Copy the sample environment file located in `env/` into the target subdirectory of your choice (currently `testnet` or `mainnet`) and prefix the filename with your blockchain of choice:
+
+```
+cp env/.env.sample env/testnet/sepolia.env
+```
```

Comment:
add a note here to create the `testnet` dir before running this command

Reply:
Whoops, I meant to create that ahead of time actually.

Reply:
better, include a mkdir command just before the cp step

Reply:
Sweet, that works.